misc/bazel/registry/modules/rules_kotlin/1.9.4-codeql.1/MODULE.bazel

```diff
@@ -0,0 +1,31 @@
+module(
```

Comment:
Can we provide attribution+License for that file in this directory, or in the registry top-level dir?

Reply:
I'm not sure where these ended up.

Reply:
no override was required any more